lib/src/note/positioned_note.dart

```diff
@@ -82,24 +82,9 @@ final class PositionedNote extends Note {
   /// Note.a.inOctave(4).equalTemperamentFrequency(338) == 338
   /// Note.bFlat.inOctave(4).equalTemperamentFrequency(338) == 464.04
   /// ```
-  double equalTemperamentFrequency([double a4Hertzs = 440]) =>
-      a4Hertzs * math.pow(sqrt12_2, Note.a.inOctave(4).difference(this));
-
-  /// Whether this [Note] is inside the human hearing range at [a4Hertzs].
-  ///
-  /// Example:
-  /// ```dart
-  /// Note.a.inOctave(4).isHumanAudibleAt() == true
-  /// Note.d.inOctave(0).isHumanAudibleAt() == false
-  /// Note.g.inOctave(12).isHumanAudibleAt(442) == false
-  /// ```
-  bool isHumanAudibleAt([double a4Hertzs = 440]) {
-    final frequency = equalTemperamentFrequency(a4Hertzs);
-    const minFrequency = 20;
-    const maxFrequency = 20000;
-
-    return frequency >= minFrequency && frequency <= maxFrequency;
-  }
+  Frequency equalTemperamentFrequency([double a4Hertzs = 440]) => Frequency(
+        a4Hertzs * math.pow(sqrt12_2, Note.a.inOctave(4).difference(this)),
```

Comment:
Whoops, and this is totally my fault:

```suggestion
  Frequency equalTemperamentFrequency([double a4Hertz = 440]) => Frequency(
        a4Hertz * math.pow(sqrt12_2, Note.a.inOctave(4).difference(this)),
```